website/server/controllers/api-v3/challenges.js

```diff
@@ -172,7 +289,13 @@ api.joinChallenge = {
  *
  * @apiSuccess {Object} data An empty object
  *
+ * @apiSuccessExample {json} Successfully left challenge
```

Comment:
Good point. I removed this and the existing examples that were just empty objects.